src/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware/Logging/HttpLoggingTagNames.cs

```diff
@@ -9,6 +9,12 @@ namespace Microsoft.AspNetCore.Diagnostics.Logging;
 /// <summary>
 /// Constants used for incoming HTTP request logging tags.
 /// </summary>
+// There is no OTel semantic convention for HTTP logs, therefore we are using
+// semantic conventions for HTTP spans:
+// https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-spans.md.
+// Some dimensions (e.g. Host and Method) were not renamed, though they had corresponding
+// names in the semantic conventions. The reason is that they are coming from ASP.NET Core 8
+// HttpLogging component, therefore we cannot rename them.
```

Comment:
Is this something we'd want end-users to see in the docs (e.g., at learn.microsoft), or is this for the maintainers only?
Ditto for the other comments.

Reply:
I didn't mean to include this comment as public documentation, and my initial intent was to include it as a reference and justification "WHY" only for maintainers. It is part of public API though, and such a comment/remark could be useful for users.

Reply:
Regarding web doc, e.g. learn.microsoft, I think that we'll include into the doc a list of log dimensions and remark that some of those dimensions are following OTel semantic-conventions.